pages/index.tsx

```diff
@@ -37,6 +46,15 @@ export default function Home() {
           </HStack>
         </VStack>
       </Container>
+      <Spacer m={4} />
+      <Image
+        m={5}
+        width={pageLogoWidth}
+        className="pageLogo"
+        src="/ssLogo.png"
+        alt="shark stewards"
+      />
```

Comment:
Is this image really necessary? Doesn't feel like it's adding much?

The page would probably look a lot better if we just made the background a hero image of sharks in the ocean! 🦈 🦈 🦈

Reply:
The image itself is not. I was trying to fill the space so that the footer doesn't come so close the text. 

When switching to light mode    the contrast and portions between the footer and main region seemed weird.

I like the hero image idea.

Reply:
A pretty clean way to handle this is just to put a `min-height` on the container instead!

Reply:
![Screenshot_20210916-183527_Chrome](https://user-images.githubusercontent.com/42547062/133694360-2dc4d6b7-6551-4cc2-bb53-de5cf1d77c91.jpg)

Reply:
Nice! I'll try that and maybe get a hero image in there.